### ISSUE TITLE
In v2.3.0 LXD default remote doesn't work

### ISSUE THREAD

**Issue by stew3254** [terraform-lxd/terraform-provider-lxd#535]
When trying to set another remote as default in LXD, I get the following error
```
│ Error: Failed initialize LXD provider
│ 
│   with provider["registry.terraform.io/terraform-lxd/lxd"],
│   on main.tf line 10, in provider "lxd":
│   10: provider "lxd" {
│ 
│ Multiple remotes are configured as default: [image-server]
```

When I change `default = true` to `default = false` it works and simply uses my local LXD instance. Here is my provider

```terraform
provider "lxd" {
  generate_client_certificates = true
  accept_remote_certificate = var.accept_remote_certificates

  remote {
    name = var.remote_image_server_name
    address = var.remote_image_server_address
    password = var.remote_image_server_password
    default = true
  }
}
```

With variable substititon, it looks something like this:
```terraform
provider "lxd" {
  generate_client_certificates = true
  accept_remote_certificate = true

  remote {
    name = mine
    address = "https://1.2.3.4:8443"
    password = "lxd"
    default = true
  }
}
```

This same config works in v2.2.0

**Comment by MusicDin** [comment]
Thanks for reporting the issue. It was fixed yesterday in https://github.com/terraform-lxd/terraform-provider-lxd/pull/534. A release will be made asap to apply the fix. 
Apologies for the inconvenience.

**Comment by MusicDin** [comment]
Fixed in 2.4.0
